canonical-web-and-design/jaas.ai
# Upgrade the cards package

## Done
- Upgrade the cards package to 1.8.0
- Deprecate the use of card sharing

## QA
- Open the demo and visit:
- /big-data and /openstack and check the card's load without a deploy button
